SampleViewer/View/ChartTrendView.swift

```diff
@@ -0,0 +1,133 @@
+import SwiftUI
+import Charts
+
+struct ChartTrendView: View {
+    // TODO: Generate daily step log
+    @State
+    private var steps: [Step] = [
+        2585,
+        2161,
+        1427,
+        3085,
+        1323,
+        2036,
+        2290,
+        3750,
+        3667,
+        1132,
+        4630,
+        1692,
+        1187,
+        1214,
+        4269,
+        4033,
+        4909,
+        4915,
+        2480,
+        1321,
+        3333,
+        1029,
+        3163,
+        2821,
+        1666,
+    ]
+    .enumerated()
+    .map { index, numberOfSteps in
+        Step(
+            id: UUID(),
+            date: Date(
+                timeIntervalSince1970: 1704207600 // 2024-01-03T00:00:00+09:00
+                + TimeInterval(60 * 60 * 24 * 7 * index) // 1 week
+            ),
+            numberOfSteps: numberOfSteps
+        )
+    }
+
+    var body: some View {
+        // TODO: Calculate daily average, not weekly average
+        let averageOfSteps =
+            steps.isEmpty ? 0 : steps.reduce(0) { $0 + $1.numberOfSteps } / steps.count
+
+        VStack {
+            Text("hig.charting-data.trend.title")
+                .font(.title)
+            Text("hig.charting-data.trend.description")
+                .font(.body)
+        }
+        .padding()
+
+        GroupBox {
+            VStack {
+                HStack {
+                    Image(systemName: "flame.fill")
+                        .foregroundStyle(Color.red)
+                    Text("hig.charting-data.trend.chart.title")
+                        .foregroundStyle(Color.red)
+                    Spacer()
+                    Image(systemName: "chevron.forward")
+                        .foregroundStyle(Color.gray)
+                }
+                .padding(.bottom)
```

Comment:
This is not `GroupBox`' `label` because so that VoiceOver can focus the same contents as Health.

|Health|this view|
|---|---|
|<img src=https://github.com/user-attachments/assets/2b2372c7-e025-4b9f-89d7-bb8b3279b57b width=200>|<img src=https://github.com/user-attachments/assets/48fdcc98-635f-4ccd-ba3e-e022889a5cbe width=200>|